tests/common/common.h

```diff
@@ -591,9 +591,9 @@ inline bool kernel_supports_wg_size(sycl_cts::util::logger& log,
     return false;
   }
 
-  sycl::program program(context, devicesToCheck);
-  program.build_with_kernel_type<kernelT>("");
-  auto kernel = program.get_kernel<kernelT>();
+  auto kb =
+      sycl::get_kernel_bundle<kernelT, sycl::bundle_state::executable>(context);
```

Comment:
@kovshikovser, it seems like we should wrap this code with `#if !defined(__COMPUTECPP__)` and keep old API in `#else` branch to fix the build with ComputeCPP. 

@psalz. am I right?

Reply:
I think so, yes!

Reply:
Actually hipSYCL doesn't have `get_kernel_bundle` either yet. The reason why passes is that hipSYCL doesn't compile `test_base_opencl.cpp`, which includes these common headers. So the check should be `#if !defined(__COMPUTECPP__) && !defined(__HIPSYCL__)`!

Reply:
Addressed in #189.